consensus/parlia/parlia.go

```diff
@@ -615,7 +616,7 @@ func (p *Parlia) VerifyUncles(chain consensus.ChainReader, header *types.Header,
 
 // Prepare initializes the consensus fields of a block header according to the
 // rules of a particular engine. The changes are executed inline.
-func (p *Parlia) Prepare(chain consensus.ChainHeaderReader, header *types.Header, ibs *state.IntraBlockState) error {
+func (p *Parlia) Prepare(chain consensus.ChainHeaderReader, header *types.Header, ibs *state.IntraBlockState, firehoseContext *firehose.Context) error {
```

Comment:
You need to update the consensus interface `Prepare` definition along with all implementation of it.